install_config/router/default_haproxy_router.adoc

```diff
@@ -444,17 +444,47 @@ corresponding to the `router=first` label.
 [[using-router-shards]]
 == Using Router Shards
 
-The access controls are based on the service account that the router is run
-with.
-
-Using
+xref:../../architecture/core_concepts/routes.html#router-sharding[_Router sharding_] uses
 xref:../../architecture/core_concepts/routes.adoc#env-variables[`NAMESPACE_LABELS`]
 and/or
 xref:../../architecture/core_concepts/routes.adoc#env-variables[`ROUTE_LABELS`],
-a router can filter out the namespaces and/or routes that it should admit.
+that enable a router to filter out the namespaces and/or routes that it should admit.
```

Comment:
This sentence confused me. Maybe:

"Router sharding uses NAMESPACE_LABELS and ROUTE_LABELS to filter router namespaces and routes."

"and/or" is against IBM (p303), and "that it should admit" is redundant (cos "filter").